openwisp_controller/config/admin.py

```diff
@@ -600,9 +601,8 @@ def clone_selected_templates(self, request, queryset):
     actions = ['clone_selected_templates']
 
 
-if not app_settings.CONFIG_BACKEND_FIELD_SHOWN:  # pragma: nocover
+if app_settings.CONFIG_BACKEND_FIELD_SHOWN is False:  # pragma: nocover
```

Comment:
I have replaced this as well, should I revert this or is it good?

Reply:
here it's not needed, we need it in the template because undefined variables default to `None`, in this case it will always be either `True` or `False`

Reply:
Done, please take another look